File: dev-docs/snippets/documents/decline-invitation.swift
import JsBaoClient

// Deprecated: no invitee-side decline verb. Swift returns an untyped
// `[String: Any]`.
func declineInvitation(
  client: JsBaoClient, documentId: String, invitationId: String
) async throws {
  // #region example
  let result = try await client.documents.declineInvitation(
    documentId: documentId, invitationId: invitationId
  )
  // #endregion example
  _ = result
}

File: dev-docs/snippets/documents/decline-invitation.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Deprecated: no invitee-side decline verb; pending invitations expire
// automatically. Decline a legacy invitation.
export async function declineInvitation(
  client: JsBaoClient,
  documentId: string,
  invitationId: string,
) {
  // #region example
  const result = await client.documents.declineInvitation(
    documentId,
    invitationId,
  );
  // #endregion example
  return result;
}
